demos/demo_robot_com_ctrl_cpp.py

```diff
@@ -122,7 +124,7 @@ def demo(robot_name):
             db,
             q[:3],
             x_com,
-            dq[:3],
+            quat.toRotationMatrix().dot(dq[:3]), # local to world frame
```

Comment:
The C++ `centrl_pd_ctrl` assumes the desired and actual com velocity to be in global frame. Therefore, rotating the base velocity from it's local (pinocchio convention) frame into the global frame here.